include/dbmanager.h

```diff
@@ -49,6 +49,12 @@ class DbManager
      * @param match : The required match.
      */
 	bool extractMatch(const int& matchId, Match& m);
+
+    bool fillMatch(const int& id,
```

Comment:
attention for indentation ;)

Reply:
My bad, I used a wrong options and I unfortunately switched between spaces and tabs indentations...
Corrected